samples/layouts/accordion/customization/src/index.tsx

```diff
@@ -1,177 +1,227 @@
-                                    })}
-                                </IgrRadioGroup>
-                            </span>
-                        </IgrExpansionPanel>
-                        <IgrExpansionPanel key="ep4">
-                            <h1 slot="title" key="ep4Title">{this.state.time}</h1>
-                            <span key="ep4Span">
-                                <IgrDateTimeInput className="size-small" key="dtInput" inputFormat="hh:mm tt" label="Arrive before"
-                                    ref={this.dateTimeInputRef} onChange={this.timeChange}>
-                                    <span key="sPrefix" slot="prefix">
-                                        <IgrIcon name="clock" collection="material" />
-                                    </span>
-                                    <span key="sSuffix" slot="suffix" onClick={this.clearTime}>
-                                        <IgrIcon name="clear" collection="material" />
-                                    </span>
-                                </IgrDateTimeInput>
-                            </span>
-                        </IgrExpansionPanel>
-                    </IgrAccordion>
+  private categories = [
+    { checked: false, type: "Bike" },
+    { checked: false, type: "Motorcycle" },
+    { checked: false, type: "Car" },
+    { checked: false, type: "Taxi" },
+    { checked: false, type: "Public Transport" },
+  ];
+
+  private dateTimeInput: IgrDateTimeInput;
+
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      categories: this.categories,
+      cost: { lower: 200, upper: 800 },
+      rating: "",
+      time: "Time",
+    };
+
+    this.categoriesChange = this.categoriesChange.bind(this);
+    this.costRangeChange = this.costRangeChange.bind(this);
+    this.ratingChange = this.ratingChange.bind(this);
+    this.timeChange = this.timeChange.bind(this);
+    this.clearTime = this.clearTime.bind(this);
+    this.dateTimeInputRef = this.dateTimeInputRef.bind(this);
+
+    registerIconFromText("clear", clearIcon, "material");
+    registerIconFromText("clock", clockIcon, "material");
+  }
+
+  public render(): JSX.Element {
+    return (
+      <div id="root">
+        <div className="sample-wrapper">
+          <IgrAccordion>
+            <IgrExpansionPanel key="ep1">
+              <h1 slot="title" key="ep1Title">
+                Categories
+                {this.state.categories.some((c: Category) => c.checked) && ": "}
+                {this.state.categories
+                  .filter((c: Category) => c.checked)
+                  .map((c: Category) => c.type)
+                  .join(", ")}
+              </h1>
+              <span key="ep1Span">
+                <div className="categories-container">
+                  {this.state.categories.map((c: Category) => {
+                    return (
+                      <IgrCheckbox
+                        key={"checkbox-" + c.type}
```

Comment:
There are a lot of instances where `key` is set. We can now remove those, since it's no longer needed.